common/src/main/resources/schema/development/schema.json

```diff
@@ -167,17 +167,123 @@
         }
       }
     },
-    "storageObject": {
+    "azureStorageObject": {
```

Comment:
I thought that it might be cleaner to separate the enums out like this, especially since we can't have comments.